content-jumpbox-provisioner/LICENCE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Do we want to put this under the MIT license?

I don't mind, I just don't remember having done it before for our other provisioners. 🙂

Reply:
I quite like open sourcing everything i write 🙈 
But if it is not the norm, i can remove it

Reply:
Nah, I don't have a problem with it, as long as you've checked whether there's any implications first. 🙂